include/libswiftnav/dgnss_management.h

```diff
@@ -22,7 +22,7 @@
 #define DEFAULT_CODE_VAR_KF     (100 * 400)
 #define DEFAULT_AMB_DRIFT_VAR   1e-8
 #define DEFAULT_AMB_INIT_VAR    1e8
-#define DEFAULT_NEW_INT_VAR     1e10
+#define DEFAULT_NEW_INT_VAR     1e4
 
```

Comment:
Done. The difference between initial and new should be deprecated soon. Now that we aren't initializing and adding sats as naively, we use the same estimator to start the KF as we do to add sats. So these variance parameters will be the same.

(But before I deprecate it, I need to make sure 1e4 is a good value for initialization too).